### PR TITLE
feat(embodied): add evac world model (2 version)

### PR DESCRIPTION
**Pull request by jzndd**
feat(embodied): add evac world model with two versions

### Description
This PR integrates the **EVAC World Model** into the RLinf framework, providing a working end-to-end instantiation that can already run successfully.  

### Motivation and Context
EVAC serves as the first fully functional example, validating the pipeline and allowing subsequent models to be dropped into the same interface.

### How has this been tested?
+ Successfully executed EVAC-based world model inside RLinf environment.
+ End-to-end rollout tested with real chunking, inference, and reward extraction.

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (Document-only update)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: rlinf/envs/worldmodel/evac/reward_model/classifier.py
Comment (zoeyuchao) [inline rlinf/envs/worldmodel/evac/reward_model/classifier.py]: We will have a separate part named reward model (using the reward model to get a reward instead of a rule-based reward). It will be integrated into ppo training pipeline. would this file the same as that one? link to PR #279

File: rlinf/envs/worldmodel/evac/sim/simulator.py [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+def get_action_bias_std(domain_name):
+    return torch.tensor(StatisticInfo[domain_name]["mean"]).unsqueeze(0), torch.tensor(
+        StatisticInfo[domain_name]["std"]
+    ).unsqueeze(0)
+
+
+def quat_to_euler(quat):
+    """
+    quat: [N, 4] or [4,] 四元数 (w, x, y, z) 或 (x, y, z, w)
+    返回: [N, 3] 欧拉角 (roll, pitch, yaw)
```
Comment (zoeyuchao) [inline rlinf/envs/worldmodel/evac/sim/simulator.py]: the chinese comment is not allowed in Rlinf. please fix them
